docroot/profiles/custom/cgov_site/modules/custom/cgov_article/config/install/language/es/core.base_field_override.node.cgov_article.promote.yml

```diff
@@ -0,0 +1,4 @@
+label: 'Promocionado a la página principal'
```

Comment:
None of the ES config was being exported before. We can discuss if this needs to be exported or not.

Reply:
Config translation is not enabled, and is a pain in the ass. Locale has a conflict where if the string in the translated config does not exist in the translated strings, then it removes the config. Come to find out, from what I can tell, translated configs that only need labels and whatnot do not require config translation...